use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResource.java

```diff
@@ -100,6 +96,50 @@ public String solve(VehicleRoutePlan problem) {
         return jobId;
     }
 
+    @Operation(summary = "Request recommendations to the RecommendedFit API for a new customer.")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "200",
+                    description = "The list of fits for the given customer.",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON,
+                            schema = @Schema(implementation = List.class)))})
+    @POST
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("recommendation")
+    public List<RecommendedFit<VehicleRecommendation, HardSoftLongScore>> recommendedFit(RecommendationRequest request) {
+        Customer customer = request.solution().getCustomers().stream().filter(c -> c.getId().equals(request.customerId()))
+                .findFirst().orElseThrow(() -> new IllegalStateException("Customer %s not found".formatted(request.customerId())));
+        List<RecommendedFit<VehicleRecommendation, HardSoftLongScore>> recommendedFitList = solutionManager
+                .recommendFit(request.solution(), customer, c -> new VehicleRecommendation(c.getVehicle().getId(),
+                        c.getVehicle().getCustomers().indexOf(c)));
+        if (!recommendedFitList.isEmpty()) {
+            return recommendedFitList.subList(0, Math.min(MAX_RECOMMENDED_FIT_LIST_SIZE, recommendedFitList.size()));
+        }
+        return recommendedFitList;
+    }
+
+    @Operation(summary = "Applies a give recommentation.")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "200",
+                    description = "The new solution updated with the recommentation.",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON,
+                            schema = @Schema(implementation = VehicleRoutePlan.class)))})
+    @POST
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("recommendation/apply")
```

Comment:
```suggestion
    @Path("recommendation")
```

And then you can use `recommendation` here, because `@POST` is no longer used to get the recommendations.

Reply:
Please see my previous comment regarding the `GET` request body.